package.json

```diff
@@ -2,11 +2,65 @@
+    "@bower_components/angular-resource": "angular/bower-angular-resource#~1.6.10",
+    "@bower_components/angular-route": "angular/bower-angular-route#~1.6.10",
+    "@bower_components/angular-sanitize": "angular/bower-angular-sanitize#~1.6.10",
+    "@bower_components/angular-scenario": "angular/bower-angular-scenario#~1.6.10",
+    "@bower_components/angular-touch": "angular/bower-angular-touch#~1.6.10",
+    "@bower_components/angular-translate": "PascalPrecht/bower-angular-translate#^2.4",
+    "@bower_components/angular-ui-notification": "MikhailRoot/angular-ui-notification#0.3.5",
+    "@bower_components/angular-ui-sortable": "angular-ui/ui-sortable#~0.13.3",
+    "@bower_components/angular-xml": "johngeorgewright/angular-xml#~2.2.1",
+    "@bower_components/angularfire": "firebase/angularFire#^2.3.0",
+    "@bower_components/bootstrap": "twbs/bootstrap#^3.0.0",
+    "@bower_components/bootstrap-chosen": "alxlit/bootstrap-chosen#*",
+    "@bower_components/bootstrap-daterangepicker": "dangrossman/bootstrap-daterangepicker#^2.0.0",
+    "@bower_components/bootstrap-sass-official": "twbs/bootstrap-sass#~3.2.0",
+    "@bower_components/chosen": "harvesthq/bower-chosen#^1.5.1",
+    "@bower_components/d3": "mbostock-bower/d3-bower#~3.5.5",
+    "@bower_components/datatables": "DataTables/DataTables#>=1.10.0",
+    "@bower_components/es5-shim": "es-shims/es5-shim#~3.1.0",
+    "@bower_components/ev-emitter": "metafizzy/ev-emitter#^1.0.0",
+    "@bower_components/file-saver.js": "Teleborder/FileSaver.js#~1.20150304.1",
+    "@bower_components/firebase": "firebase/firebase-bower#3.x.x",
+    "@bower_components/fontawesome": "FortAwesome/Font-Awesome#~4.4.0",
+    "@bower_components/google-diff-match-patch-js": "tanaka-de-silva/google-diff-match-patch-js#^1.0.0",
+    "@bower_components/imagesloaded": "desandro/imagesloaded#>=3.0.0",
+    "@bower_components/jquery": "jquery/jquery-dist#~2.1.1",
+    "@bower_components/jquery-autosize": "jackmoore/autosize#^4.0.2",
+    "@bower_components/jquery-ui": "components/jqueryui#>=1.9",
+    "@bower_components/js-xlsx": "SheetJS/js-xlsx#~0.7.11",
+    "@bower_components/json3": "bestiejs/json3#~3.3.1",
+    "@bower_components/jspdf": "MrRio/jsPDF#~1.0.272",
+    "@bower_components/jszip": "Stuk/jszip#~2.4.0",
+    "@bower_components/levenshtein": "gf3/Levenshtein#~1.0.5",
+    "@bower_components/lodash": "lodash/lodash#~4.17.11",
+    "@bower_components/moment": "moment/moment#>=2.9.0",
+    "@bower_components/ng-tags-input": "mbenford/ngTagsInput-bower#3.2.0",
+    "@bower_components/oridomi": "dmotz/oriDomi#~1.1.5",
+    "@bower_components/please-wait": "Pathgather/please-wait#~0.0.2",
+    "@bower_components/spinkit": "tobiasahlin/SpinKit#~1.0.1",
+    "@bower_components/stringjs": "jprichardson/string.js#~2.2.0",
+    "@bower_components/uuid-js": "aurigadl/uuid-js#^0.7.5",
+    "@bower_components/x2js": "abdmob/x2js#~1.2.0"
```

Comment:
why are the components with bower prefix?

Reply:
I followed the official bower removal documentation. It introduces a bower-away package.
https://bower.io/blog/2017/how-to-migrate-away-from-bower
node_modules/@bower_components will contain all components in exactly the same way they would be installed by Bower

Reply:
@victoria34 should we use npm packages?

Reply:
I cannot find all corresponding bower packages in npm. If we really want to switch to NPM thoroughly, we may need to use some npm packages to replace bower components, fix version compatibility issue and rewrite code based on bower.

Reply:
@victoria34 it's not urgent, as long as the packages are under npm, I'm ok with that. Let's leave it like this for now.

Reply:
Thank you